utilities/private/sn2individual.m

```diff
@@ -86,8 +86,7 @@
   V.descrip = 'dummy volume';
   create(V);
   V.dat(:) = 0;
-  P.image(1).private = V;
-  P.image(1).fname   = fname;
+  P.image(1) = spm_vol(fname);
```

Comment:
Can you provide a test function (or add your faulty use case to an existing test function) to demonstrate the error in your particular use case, and to ensure that other (hencetoforth) non-erroneous use cases still work with the suggested changes in place? Thanks.

Reply:
I specifically wonder if this is SPM version specific, with "SPM version" as in a subtle difference that FT is not aware of or sensitive to. Like from SPM12.1 to SPM12.2, if such versions would exist.